e2e/cypress/integration/onboarding/enter_support_email_spec.js

```diff
@@ -7,15 +7,13 @@
 // - Use element ID when selecting an element. Create one if none.
 // ***************************************************************
 
-// Group: @enterprise @onboarding @cloud_only
+// Group: @onboarding
 
-describe('Cloud Onboarding - Sysadmin enter support email', () => {
+describe('Onboarding - Sysadmin enter support email', () => {
```

Comment:
I just noticed this spec file. All new tests should have Zephyr test ID in Cypress so that results will be recorded in Zephyr -- for reporting and tracking. Could you please add all `it` blocks with Zephyr ID? You may ask your team QA to help you validate the test criteria and eventually be added to Zephyr. Thanks!